Frontend/ui-common/src/Engineering/Helpers/Commons.js

```diff
@@ -459,3 +459,8 @@ export const getImageUrl = function (url) {
     console.log("error in getImageUrl " + e);
   }
 };
+
+export const isTrue = function (a) {
```

Comment:
Rebase PR with main and Use toBool function present in mobilityPrelude

Reply:
toBool is not working with framework callback boolean in this case, so created this function